pkg/ccl/jwtauthccl/authentication_jwt_test.go

```diff
@@ -0,0 +1,333 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package jwtauthccl
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/base"
+	"github.com/cockroachdb/cockroach/pkg/security/username"
+	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
+	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/lestrrat-go/jwx/jwa"
+	jwk "github.com/lestrrat-go/jwx/jwk"
+	"github.com/lestrrat-go/jwx/jwt"
+	"github.com/stretchr/testify/require"
+)
+
+var username1 = "test_user1"
+var username2 = "test_user2"
```

Comment:
this is hitting the unused variable linter:
```
the following objects are unused: github.com/cockroachdb/cockroach/pkg/ccl/jwtauthccl.username2
```

Reply:
Yeah. When I rebased my local state now needs to rebuild a lot so I got ahead of myself. I expect a number of small linter/etc style fixes will be needed (example: adding a test timeout).